tests/telemetry/conftest.py

```diff
@@ -58,7 +58,7 @@ def check_gnmi_config(duthost):
 
 
 def create_gnmi_config(duthost):
-    cmd = "sonic-db-cli CONFIG_DB hset 'GNMI|gnmi' port 50052"
+    cmd = "sonic-db-cli CONFIG_DB hset 'GNMI|gnmi' port 8080"
```

Comment:
Out of curiosity, may I know why we are not using the old port `50052` anymore?

Reply:
The 202405 and master uses 8080 for telemetry inside gnmi by default. 
ps -ef | cat
/usr/sbin/telemetry -logtostderr --noTLS --port 8080 --allow_no_client_auth -v=2 --threshold 100 --idle_conn_duration 5

Reply:
In 202205 it uses 50051.

Reply:
OK. I can change it back to 50052.